app/js/header5.js

```diff
@@ -0,0 +1,44 @@
+var touches = document.querySelectorAll('.fifth__touch');
+var left = document.querySelector('.fifth__arrow--left');
+var right = document.querySelector('.fifth__arrow--right');
+var number = 0;
+
+for(var i = 0; i < touches.length; i++) {
+    touches[i].addEventListener('click', Click(i));
+}
+
+function Click(index) {
```

Comment:
in camelCase you should write small first letter so your function will be called `click`.
And you don't need to write this `return function`. You can write just 
```javascript
function click(index) {
  number = index;
  clearClasses();
  this.classList.add('fifth__touch--active');
})
```

Reply:
This's JavaScript - if I do it like you it won't work.
Sorry, I don't explain you this stuff now, because I don't have much time, but you can search this in google.

Reply:
okey, I got it. Try
```javascript
function click(function(index){
  number = index;
  clearClasses();
  this.classList.add('fifth__touch--active');
})
```

Reply:
Nope (
Nesting function don't help if we have arrays.
I know only solution with `return`.
If you anything else, write me and I will change it.

Reply:
okay, you can now leave this function in working condition and on the lesson I'll suggest you another algorithm how to handle active state of slides in easier way